postgres/datadog_checks/postgres/postgres.py

```diff
@@ -594,6 +599,11 @@ def _query_scope(self, cursor, scope, key, db, instance_tags, relations, is_cust
             # metric-map is: (dd_name, "rate"|"gauge")
             # shift the results since the first columns will be the "descriptors"
             values = zip([scope['metrics'][c] for c in cols], row[len(desc):])
+            # keep sending the deprecated metrics if any
+            deprecated_values = zip([scope.get('deprecated_metrics', {}).get(c) for c in cols], row[len(desc):])
+            for metric_map, value in deprecated_values:
+                if metric_map is not None:
```

Comment:
Do we want to emit some type of deprecation notice/warning when we get here? Even if its just the first time?

Reply:
I'd like to define a global process before, for now we could say it's not even deprecated, we just send a new metric